pkg/app/piped/controller/planner.go

```diff
@@ -325,3 +342,21 @@ func (p *planner) reportDeploymentCancelled(ctx context.Context, commander, reas
 	}
 	return err
 }
+
+func (p *planner) getMentionedAccounts(ctx context.Context, event model.NotificationEventType, targetDSP deploysource.Provider) ([]string, error) {
+	ds, err := targetDSP.GetReadOnly(ctx, ioutil.Discard)
```

Comment:
nit: the `ioutil` package got deprecated. Let's use `io.Discard` instead.

Reply:
All functions and global variables are mapped to other packages. So be proactive in using them for newly added code.

Reply:
6544bbf